lessons/2-shell/slides.qmd

```diff
@@ -1,166 +1,177 @@
 logo: "../../logo.png"
 ---
 
-# Review of last session
+## Review of last session
 
 -   What questions do you have?
 -   What happened as you were trying to re-organize projects or R code?
     -   Victories?
     -   Stumbling blocks?
 
-# Learning objectives
+## Learning objectives
 
--   Understand the what and why of the shell
--   Use the shell to navigate your computer's file system
--   Use R projects to simplify file system navigation
+-   Understand your computer's *file system*
+-   Understand the what and why of the shell command-line interface
+-   Use the CLI to navigate your computer's file system
 -   Know your options for finding help
 
-# What is the shell?
+## How well do you know your file system?
 
--   Command-line interface, in contrast to a GUI
+## Example file system
+
+![](file-path-example.png)
 
-# Why learn the shell?
+## Example file system
+
+![](file-structure-example.png)
 
--   Be able to work with tools that don't have a GUI
--   Batch large tasks (e.g. renaming files)
--   Interact with remote computers like the University HPC
+## How do you navigate your file system?
 
-# Shell in this workshop
+-   Searching
+-   Point and click
+-   "Go to path"
+
+## How do we navigate the file system *programmatically*?
+
+-   A script can't point and click!
+-   We need a system for moving around our file system through written commands.
+
+## What is the shell?
+
+-   Command-line interface, in contrast to a GUI
+-   Useful for navigating the file system
+-   ...and for automating large tasks (e.g. renaming, moving files)
+-   ...*and* for running programs that may not have a GUI
+-   ...**and** for interfacing with remote computers (e.g. the HPC)
+
+## Shell in this workshop
 
 -   git/GitHub (next 2 lessons)
 -   Navigating your file system
 -   Quarto (lesson 9)
 
-# Let's get set up!
+## Let's get set up!
 
 -   macOS comes with a shell (usually Bash or Zsh) built-in
 
 -   Windows need to install GitBash (includes Bash shell and git, which we will use later)
 
 -   Breakout rooms
 
-    -   Mac users go with Kristina
+    -   Mac users go with Eric
     -   Windows users go with Renata
 
-# Data download
+## Data download
 
 -   Download `shell-lesson-data.zip`
     -   Instructions here: <https://swcarpentry.github.io/shell-novice/index.html#download-files>
 -   Save it to your Desktop and unzip it.
 
-# Coding time
+## Coding time
 
 -   RStudio panes
 -   Set up Terminal in RStudio
 -   Navigation using Terminal and Files pane
 
-# Example file system
-
-![](file-path-example.png)
-
-# Example file system
-
-![](file-structure-example.png)
-
-# Syntax of a shell command
+## Syntax of a shell command
 
 ![](shell_syntax.png){fig-align="center"}
 
-# Coding time
+## Coding time
 
 -   Synchronize working directories between Terminal, Files Pane, and R working directory
 
-# Coding time
+## Coding time
 
 -   Use an R project to simplify working directories
```

Comment:
```suggestion
-   Use an RStudio project to simplify working directories
```
I didn't remember that we introduced RStudio projects here! That's great new for me, since I was thinking I'd have to cram that into lesson 1.  Now that I look back over the slides and notes, this does seem like a natural place to introduce them.

Reply:
Ha, I actually was trying to dial this down in this lesson because it looked like you were going to cover it in lesson 1! But it fits neatly here. Since there's already so much content in lesson 1, maybe we can mention projects in lesson 1 in passing, as foreshadowing, and then actually teach it in this lesson?

Reply:
Decided to keep this in, but suggested change from "R project" to "RStudio project" still stands